doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -249,83 +253,93 @@ Induction
    is **IH**.  The tactic is similar to :tacn:`destruct`, except that
    `destruct` doesn't generate induction hypotheses.
 
-   There are several cases:
+   :n:`induction @one_term ...` also works when the type of :n:`@one_term` is a statement
+   with premises and whose conclusion is inductive. In that case the tactic
+   performs induction on the conclusion of the type of :n:`@one_term` and leaves the
+   :term:`non-dependent premises <dependent premise>` of the type as subgoals.
+   In the case of dependent products, the tactic tries to find an instance for
+   which the induction principle applies and otherwise fails.
+
+   Most parameters behave as they do in :tacn:`destruct`, with the following
+   differences:
```

Comment:
There might be a misunderstanding. What is described below do not represent differences with `destruct`. On the contrary, `destruct` interprets its main argument in the same way.

Reply:
I struck out the part about differences.  Are we good now?